cr8/clients.py

```diff
@@ -3,6 +3,22 @@
 import itertools
 from datetime import datetime
 from typing import List, Union, Iterable
+from decimal import Decimal
+
+class CrateJsonEncoder(json.JSONEncoder):
+
+    epoch = datetime(1970, 1, 1)
+
+    def default(self, o):
+        if isinstance(o, Decimal):
+            return str(o)
+        if isinstance(o, datetime):
+            delta = o - self.epoch
+            return int(delta.microseconds / 1000.0 +
```

Comment:
I think this can be changed to `int(delta.total_seconds() * 1000)`.  (total_seconds wasn't available in earlier python versions, that's why it isn't used in crate-python)